.gitignore

```diff
@@ -53,3 +53,4 @@ celerybeat-schedule
 # compiled translations
 /locale/*/*/*.mo
 .grunt
+node_modules/
```

Comment:
lol saw this and had no idea how i could've missed this, but i think i have node_modules in my global ignore. good catch